src/test/java/org/folio/rest/impl/UsersAPIIT.java

```diff
@@ -121,8 +121,28 @@ static void postTenant() {
     given().when().get("/_/tenant/" + id).then().statusCode(200).body("$", not(hasKey("error")));
   }
 
-  void facets(int limit) {
+  static void userExists(String id) {
+    given().
+    when().get("/users/" + id).
+    then().statusCode(200);
+  }
+
+  static void userDoesntExist(String id) {
+    given().
+    when().get("/users/" + id).
+    then().statusCode(404);
+  }
+
+  static void postUser(String id, String username) {
+    given().
+    when().
+      body(new JsonObject().put("id",  id).put("username", username).encode()).
+      post("/users").
+    then().
+      statusCode(201);
+  }
```

Comment:
Why do these need to be static?

Personally, I think the tests are the star of the show and any helpers (which I assume these are) are intended to make the tests more readable are better kept out of the way at the bottom of the file.

Reply:
The static keyword indicates that the method doesn't use any instance variables, and it can be used in the static `@BeforeAll` and `@AfterAll` methods if at some time a need arises.

Reply:
The new helper methods are put at the same place where the existing helper method facets(int) already exists for consistency.

If you want the order been changed from "helpers - tests" to "tests - helpers" please create an issue so that it can be handled separately.

Reply:
> The static keyword indicates that the method doesn't use any instance variables, and it can be used in the static @BeforeAll and @AfterAll methods if at some time a need arises.

Why preemptively mark them as static if it's not needed now?

> If you want the order been changed from "helpers - tests" to "tests - helpers" please create an issue so that it can be handled separately.

I think that is an unnecessary overhead and is unlikely to lead to these changes being made.

Reply:
> Why should I declare the new helper methods to be non-static when they in fact are also static? Isn't this inconsistent and misleading?

Why would it having a method that doesn't need to be static, be non-static be misleading?

Personally, I think methods should be declared static when they need to be. If they aren't used in a static context, I don't think they need to be static.

I would also put static helper methods below the tests (which for me are the purpose of the class).

However, I understand that for most folks static methods belong at the top of the class (and that overrides the prominence of the tests for them) and they prefer to put methods close to where they are used.

The three examples you cite are for the purposes of shared setup. They need to be static because they are used in a static context.  I don't think all helper methods must be static by definition, nor is there is value in consistency with those only for the sake of it.